test/user.test.js

```diff
@@ -2442,7 +2442,7 @@ describe('User', function() {
     it('handles subclassed user with no accessToken relation', () => {
       // setup a new LoopBack app, we don't want to use shared models
       app = loopback({localRegistry: true, loadBuiltinModels: true});
-      app.set('_verifyAuthModelRelations', false);
+      app.set('_warnOnBadAuthModelsSetup', false);
```

Comment:
> it would mean that it's the exact same name as the app method

That shouldn't matter, because the flag is stored in a config hash-map.

> the rationale for this change is: the _verifyAuthModelRelations method, for a same name, has now a larger scope than the equally named flag

I see your point, let's make the proposed change then.